static/tag/workspace/workspace-left-menu-item.tag

```diff
@@ -0,0 +1,52 @@
+<workspace-left-menu-item>
+  <a href={opts.href}>
+    <img src={opts.icon} alt={opts.tooltip} title={opts.tooltip}>
+    <p>{opts.label}</p>
+    <div class="arrow-left"></div>
+  </a>
+
+  <style>
+    :scope {
+      flex-basis: 100px;
+      flex-grow: 0;
+      transition: all 250ms ease-in-out;
+    }
+
+    :scope:hover, :scope[active] {
+      background-color: rgb(137, 208, 245);
```

Comment:
I am not sure about the color, however, for UX reason, we should add a hover so the user know which menu item s.he is clicking.